asidi-bsas.md

```diff
@@ -0,0 +1,16 @@
+# Buenos Aires, Arg
+
+### Restaurants
+
+- McDonalds
+
+
+### Attractions for Tourists
+
+- Tango
+- Barbecue
+
```

Comment:
Nice Elisa! there are lots of lizards over there 🐊  (and not in a good mood precisely )